docs/docker.md

```diff
@@ -179,7 +179,7 @@ for detailed info on sbt mappings.
 
 ### dockerImageBase
 Value type: `String`  
-Default: `allenai-docker-private-docker.bintray.io/oracle-java:8`
+Default: `allenai-sbt-plugins-default-registry.allenai.org/oracle-java:8`
```

Comment:
Is this an actual URL or a placeholder example?

Reply:
It's a placeholder. The doc in the DockerBuildPlugin object is updated to be: "Must be overridden to push images.".

I can make this value even more explicitly fake: `allenai-sbt-plugins-FAKE-REGISTRY.allenai.org`

Reply:
I would make it more explicitly fake.